nexus/tests/unit/test_machines.py

```diff
@@ -1269,18 +1269,30 @@ def job_commands_equal(c1,c2):
         ('kagayaki'       , 'n2_t2'         ) : 'mpirun -machinefile $PBS_NODEFILE -np 128 -x OMP_NUM_THREADS test.x',
         ('kagayaki'       , 'n2_t2_e'       ) : 'mpirun -machinefile $PBS_NODEFILE -np 128 -x OMP_NUM_THREADS test.x',
         ('kagayaki'       , 'n2_t2_p2'      ) : 'mpirun -machinefile $PBS_NODEFILE -np 4 -x OMP_NUM_THREADS test.x',
+        ('lassen'         , 'n1'            ) : 'lrun -M "-gpu" -N 1 -T 42 test.x',
+        ('lassen'         , 'n1_p1'         ) : 'lrun -M "-gpu" -N 1 -T 1 test.x',
+        ('lassen'         , 'n2'            ) : 'lrun -M "-gpu" -N 2 -T 42 test.x',
+        ('lassen'         , 'n2_t2'         ) : 'lrun -M "-gpu" -N 2 -T 21 --threads=2 test.x',
+        ('lassen'         , 'n2_t2_e'       ) : 'lrun -M "-gpu" -N 2 -T 21 --threads=2 test.x',
+        ('lassen'         , 'n2_t2_p2'      ) : 'lrun -M "-gpu" -N 2 -T 2 --threads=2 test.x',
+        ('ruby'           , 'n1'            ) : 'srun test.x',
+        ('ruby'           , 'n1_p1'         ) : 'srun test.x',
+        ('ruby'           , 'n2'            ) : 'srun test.x',
+        ('ruby'           , 'n2_t2'         ) : 'srun test.x',
+        ('ruby'           , 'n2_t2_e'       ) : 'srun test.x',
+        ('ruby'           , 'n2_t2_p2'      ) : 'srun test.x',
         ('kestrel'        , 'n1'            ) : 'srun test.x',
         ('kestrel'        , 'n1_p1'         ) : 'srun test.x',
         ('kestrel'        , 'n2'            ) : 'srun test.x',
         ('kestrel'        , 'n2_t2'         ) : 'srun test.x',
         ('kestrel'        , 'n2_t2_e'       ) : 'srun test.x',
         ('kestrel'        , 'n2_t2_p2'      ) : 'srun test.x',
-        ('inti'           , 'n1'            ) : 'mpirun -np 64 test.x',
-        ('inti'           , 'n1_p1'         ) : 'mpirun -np 1 test.x',
-        ('inti'           , 'n2'            ) : 'mpirun -np 128 test.x',
-        ('inti'           , 'n2_t2'         ) : 'mpirun -np 64 test.x',
-        ('inti'           , 'n2_t2_e'       ) : 'mpirun -np 64 test.x',
-        ('inti'           , 'n2_t2_p2'      ) : 'mpirun -np 4 test.x',
+        ('inti'           , 'n1'            ) : 'srun test.x',
```

Comment:
Switch of mpirun -> srun was intentional for inti?

Reply:
Yes